keps/sig-storage/3333-reconcile-default-storage-class/kep.yaml

```diff
@@ -0,0 +1,51 @@
+title: Retroactive default StorageClass assignement
+kep-number: 3333
+authors:
+  - "@RomanBednar"
+owning-sig: sig-storage
+participating-sigs:
+status: provisional
+creation-date: 2022-06-03
+reviewers:
+  - "@jsafrane"
+  - "@xing-yang"
+approvers:
+  - TBD
+
+##### WARNING !!! ######
+# prr-approvers has been moved to its own location
+# You should create your own in keps/prod-readiness
+# Please make a copy of keps/prod-readiness/template/nnnn.yaml
+# to keps/prod-readiness/sig-xxxxx/00000.yaml (replace with kep number)
+#prr-approvers:
+
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.25"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.25"
```

Comment:
You will need to add a PRR approval file in:
https://github.com/kubernetes/enhancements/tree/master/keps/prod-readiness/sig-storage

Feel free to add me a PRR approver.

Reply:
Thanks, added.